test/test_compat.py

```diff
@@ -43,7 +43,7 @@ def test_preprocess_query():
     assert "SELECT * FROM foo WHERE id = 123" == preprocess_query(
         "SELECT * FROM foo WHERE id = 123"
     )
-    assert "SELECT /* foo */ test FROM foo.bar" == preprocess_query(
+    assert "SELECT /* foo */ test\nFROM `foo`.`bar`" == preprocess_query(
         "SELECT /* foo */ test\nFROM `foo`.`bar`"
```

Comment:
👌🏻

Reply:
How can we now get the query without comments and line breaks?

Reply:
I though it's only internal or you generalize 🤣 (but good point)

I can change the `query` property to return no comments and without line breaks as it can be used just externally. 

Or if you prefer to keep the `query` with those the query can only remove new lines and without comments should remove comments. 

Before the change it was either without new lines (`query`), or without comments (as `without_comments` uses `_raw_query` not the query (so before preprocessing)). You also couldn't get both?

Reply:
@macbre So let me know if you want to remove comments when accessing `query` property or not? :)